dv/uvm/core_ibex/ibex_dv.f

```diff
@@ -7,6 +7,7 @@
 +define+BOOT_ADDR=2147483648 // 32'h8000_0000
 +define+TRACE_EXECUTION
 +define+RVFI
++define+DV_FCOV
```

Comment:
What is the need for adding this AND the plusarg? In general runtime controls should be sufficient.

Reply:
It's for the FCOV macros that are used in design files. If it's not defined then the those macros don't produce anything. This is useful in things like synthesis runs or formal verification or just plain simulation where you don't want the extra fcov_ signals cluttering things up.

Reply:
Why not protect them with `` `ifdef SYNTHESIS`` instead?

Reply:
As I said synthesis is one reason you'd want to exclude them, but there may be others. It seemed reasonable to use a new define to control whether or not they are included. It's similar to the `INC_ASSERT` define.

Reply:
`INC_ASSERT` does not need to be passed externally. It is defined based on the toolflow. 
https://github.com/lowRISC/opentitan/blob/577bcb8364eb792edad5d096868476f7054a8249/hw/ip/prim/rtl/prim_assert.sv#L72

Perhaps we could follow the same approach here and define `INC_FCOV` inline also based on the toolflow? I prefer to keep the list of macros that need to be defined to enable / disable features to a minimum.

Reply:
I've updated things to use an `EXCLUDE_FCOV` define instead. So the FCOV stuff is in by default and we've got some tool specific `ifdef`s in the header to exclude it for synthesis.

I did it this way round as the situations you might not want it could be quite specific so preferred an option for people to remove it as part of setting up options for their flow rather than having to make modifications to dv_fcov_macros.svh.